### PR TITLE
API 2.0.0

### PR DESCRIPTION
**Pull request by SirFancyBacon**
Starting the work to migrate to API 2.0.0. definitely don't want to merge this just yet.

But! i am welcoming some guidance as to what is likely wrong with this :)

### CODE REVIEW

File: thirdparty-config/classes/Race-ExRace.conf [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+#Description displayed in game; must be a string.
 Description: "Describe your race here"
-#the Name of the class files which you want your race to beable to use, must match the Name: variable within the <classname>.conf file.
-AllowedClasses: [
-  ExClass
+AttributePointsPerLevel: 0
+SkillPointsPerLevel: 1
+#see the wiki for more information about leveling.
+Leveling: {
+    __class__: cz.neumimto.rpg.players.leveling.Exponential
```
Comment (NeumimTo) [inline thirdparty-config/classes/Race-ExRace.conf]: cz.neumimto.rpg.players.leveling.Exponential should be "cz.neumimto.rpg.players.leveling.Exponential"

File: thirdparty-config/classes/Race-ExRace.conf [hunk 2]
```diff
@@ -1,10 +1,35 @@
-#Name must be unique betweem races, classes, and skilltrees
-Name: ExRace
-MenuIcon: "minecraft:gold_ore"
+# More indepth information can be found on the Ore wiki for NT-RPG
+#
+#"Name" must a string and must be unique betweem races, classes, and skilltrees
+"Name": "ExRace"
+#"ClassType" must be a string and must be defined in the settings.conf file
+"ClassType": "Race"
```
Comment (NeumimTo) [inline thirdparty-config/classes/Race-ExRace.conf]: "ClassType": "Race" should be 
ClassType: Race

File: thirdparty-config/classes/Race-ExRace.conf [hunk 3]
```diff
@@ -1,10 +1,35 @@
-#Name must be unique betweem races, classes, and skilltrees
-Name: ExRace
-MenuIcon: "minecraft:gold_ore"
+# More indepth information can be found on the Ore wiki for NT-RPG
+#
+#"Name" must a string and must be unique betweem races, classes, and skilltrees
+"Name": "ExRace"
```
Comment (NeumimTo) [inline thirdparty-config/classes/Race-ExRace.conf]: "Name": "ExRace" 
To
Name: ExRace

**Review by NeumimTo — CHANGES_REQUESTED** [review]

File: thirdparty-config/classes/Race-ExRace.conf (hunk 1 above)
Comment (SirFancyBacon) [reply thirdparty-config/classes/Race-ExRace.conf]: Fixed

File: thirdparty-config/classes/Race-ExRace.conf (hunk 2 above)
Comment (SirFancyBacon) [reply thirdparty-config/classes/Race-ExRace.conf]: Fixed

File: thirdparty-config/classes/Race-ExRace.conf (hunk 3 above)
Comment (SirFancyBacon) [reply thirdparty-config/classes/Race-ExRace.conf]: Fixed.

**Comment by SirFancyBacon** [comment]
Okay you should be all set.
Review when you have time to make sure it's up to snuff.

**Pull request merged**